frontend/src/components/task/task.js

```diff
@@ -62,6 +67,9 @@ import TaskAssigned from './task-assigned'
 import TaskInvite from './task-invite'
 import TaskLabels from './task-labels'
 import TaskLevel from './task-level'
+const taskCover = require('../../../public/images/task-cover.png')
```

Comment:
```suggestion
const taskCover = require('../../images/task-cover.png')
```
And the same for the others